backend/domain/Room.js

```diff
@@ -1,6 +1,8 @@
 const GameStateEnum = require('./Enum/GameStateEnum');
```

Comment:
mostly not part of this PR, ignore (rebase stuff urgh)

the last part about uuids is part of this PR.